ddtrace/internal/log_writer.py

```diff
@@ -72,7 +72,6 @@ def enqueue(self, log):
             self._buffer.append(log)
 
     def on_shutdown(self):
-        # type: (...) -> None
```

Comment:
Should we revert this change?

Reply:
This is a slightly bigger issue than type hinting - mypy (which isn't enabled for contrib files I believe, so it's only flagging it now that we've moved this to `internal`) flags this as the superclass method is actually a static method (so no reference to `self`) and thus has a different function signature `# type: () -> Any`. 

I think we should probably address this through a redesign, but that would be a lot of work and unnecessary for the scope of this PR. I made the decision to ignore this for now and address it as a future PR.